py3status/modules/bluetooth.py

```diff
@@ -3,84 +3,109 @@
 Display bluetooth status.
 
 Configuration parameters:
-    cache_timeout: how often we refresh this module in seconds (default 10)
-    device_separator: the separator char between devices (only if more than one
-        device) (default '|')
-    format: format when there is a connected device (default '{name}')
-    format_no_conn: format when there is no connected device (default 'OFF')
-    format_no_conn_prefix: prefix when there is no connected device
-        (default 'BT: ')
-    format_prefix: prefix when there is a connected device (default 'BT: ')
+    always_collapse: always collapse to hide format_bt (default False)
+    button_toggle: mouse button to toggle visibility (default 1)
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default 'BT: {format_bt}')
+    format_bt: display format for bluetooth (default '{name}')
+    string_separator: show separator only if more than one (default '|')
```

Comment:
Can we just keep `device_separator` for now.  That should speed up getting things merged.